### PR TITLE
(GH-496) Prepare for 0.18.0 release

### LINKED ISSUE #496: Release 0.18.0 of VSCode
Release 0.18.0 of VSCode

### PR DESCRIPTION
**Pull request by glennsarti**
This commit prepares the extension for a 0.18.0 release.

Fixes #496

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -319,4 +334,6 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 - Initial release of the puppet extension.
 
+[0.18.0]: https://github.com/lingua-pupuli/puppet-vscode/compare/0.18.0..0.17.0
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: Need an entry for `Unreleased`

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,21 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## Unreleased
```
Comment (michaeltlombardi) [inline CHANGELOG.md]: Should be wrapped in square brackets

File: CHANGELOG.md (hunk 1 above)
Comment (michaeltlombardi) [reply CHANGELOG.md]: The versions here and the line below should be flipped.

Comment (glennsarti) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (glennsarti) [reply CHANGELOG.md]: Fixed.

**Pull request merged**